EKF/mag_fusion.cpp

```diff
@@ -623,14 +623,14 @@ void Ekf::fuseHeading()
 		}
 	}
 
-	// wrap the heading to the interval between +-pi
-	measured_hdg = wrap_pi(measured_hdg);
```

Comment:
Iirc the previous implementation did ((y mod pi) - x) mod pi, which isn't the same as what I did here. I don't think it's broken, it's just not what is needed here.